web/player.ts

```diff
@@ -209,37 +181,52 @@ export default class Player {
 
     this.scene = scene;
     this.renderer = renderer;
-    this.loop = loop;
     this._scale = scale;
+    this.frameRate = frameRate;
 
+    this.paths = paths
+    this.playMode = playMode || PlayModeEnum.Loop
+    this.prevMeshBufferHasEnough = false
+    
+    //create video element
     this._video = video ? video : createElement('video', {
       crossorigin: "",
       playsInline: "true",
       preload: "auto",
-      style: {
-        display: "none",
-        position: 'fixed',
-        zIndex: '-1',
-        top: '0',
-        left: '0',
-        width: '1px'
-      },
-      playbackRate: 1
+      playbackRate: 1,
+      muted: "muted",
     });
     this._video.setAttribute('crossorigin', '');
     this._video.setAttribute('preload', 'auto');
+    this._video.setAttribute('playsInline', 'playsInline');
```

Comment:
playsinline attribute should be all lowercase, and the value should be an empty string